includes/wikia/tasks/Tasks/BatchRefreshLinksForTemplate.class.php

```diff
@@ -83,12 +83,20 @@ public function readyRefreshLinksForTitleTask( \Title $title ) {
 		$task->title( $title );
 		$task->call( 'refresh' );
 		$task->wikiId( $this->getWikiId() );
+
+		// TODO: delay the tasks - see PLATFORM-1192
+
```

Comment:
Let's try the current approach and introduce the delay for these tasks when the current fix is not enough.